core/src/test/resources/sql/unnest.iq

```diff
@@ -223,4 +223,25 @@ FROM UNNEST(array [0, 2, 4, 4, 5]) as x;
 
 !ok
 
+
+WITH tab(field1, field2) AS (VALUES 
+	(1, ARRAY[ ROW(0), ROW(NULL)]), 
```

Comment:
Why do you need the ROW constructor inside the array? It won't work if it is simple `ARRAY[0, null]`?